netket/_scipy_integrators.py

```diff
@@ -0,0 +1,131 @@
+import scipy
+import numpy as np
+from scipy.integrate import OdeSolver, DenseOutput
+from warnings import warn
+
+
+def validate_first_step(first_step, t0, t_bound):
```

Comment:
```suggestion
def _validate_first_step(first_step, t0, t_bound):
```
(And corresponding changes below.)